src/libmongoc/src/mongoc/mongoc-server-description-private.h

```diff
@@ -184,7 +184,7 @@ void
 mongoc_server_description_set_topology_version (mongoc_server_description_t *sd,
                                                 const bson_t *tv);
 
-/* If a service_id is set on the topology description, copies it to @oid and
+/* If a service_id is set on the server description, copies it to @oid and
```

Comment:
Good eye!

Reply:
Skipped this commit while rebasing as I saw you removed the comment in https://github.com/mongodb/mongo-c-driver/commit/635be2f7b105cdd62ae95c045b436f8837f2be83#diff-de5aea95eea3d2b786d33b8b694d349c445847e9cc506502c6824e05f177cc0cL187